### DISCUSSION TITLE
Multiseat support

### DISCUSSION THREAD

**Discussion by psychrabi** [rustdesk/rustdesk#6151] [Feature Request]
We have multiseat environment in our place and currently rustdesk gives same Id in all workplaces.
This has forced us to use Anydesk without install or Parsec for remote access to workplaces.

Would love to see RustDesk support multiseat environment by giving each workplace/user its own ID so it can be indivdually accessed remotely

**Comment by My1** [comment]
what do you specifically mean with "multiseat"? a Terminal Server?

yeah seems like TS. it only seems to be able to connect to console currently

**Comment by psychrabi** [comment]
Our multseat program is called ASTER and have 1000s of users. Our software creates multiple workplaces from one system.
https://ibiksoft.com/

We would like to offer RustDesk for remote access to each workplaces
however, currently thats not possible
and have recommended our customers to use Anydesk or Parsec

**Reply by My1** [reply]
multiple workplaces on one PC so a bit like terminal server okay.

**Comment by psychrabi** [comment]
Anydesk works fine as long as its not installed, if installed, it will show same address in all workplaces.
Parsec can be installed as per user installation and allow to login to different user accounts on each workplaces, there by allowing remote to individual workplaces.

If rustdesk can achieve this, you will get more customer base for RustDesk

**Reply by My1** [reply]
installed anydesk ideally (at least in modern versions) should show on terminal-server (and possibly similar setups too) a thing in the form of `123456789/domain/user.name` at least from our experience.

**Reply by psychrabi** [reply]
https://www.youtube.com/watch?v=PnwIQ8Q35bA
Refer this video, on how Anydesk works on workplaces created by our ASTER program.

It would be great if rustdesk can work like Anydesk in workplaces.

**Reply by My1** [reply]
interesting that anydesk only works if not installed as it is terminal aware normally, so at least on a semi-proper TS enviromnent an installed anydesk would let you choose the session

**Reply by psychrabi** [reply]
Yes. So, is it possible for RustDesk to work similar to Anydesk ?
in multiseat environment

You can install ASTER and configure few workplaces and try it and see
https://ibiksoft.com/download-aster-multiseat-software

The video in the page shows how to create workplaces using ASTER.
